### DISCUSSION TITLE
TypeError: (0 , wagmi__WEBPACK_IMPORTED_MODULE_12__.createClient) is not a function

### DISCUSSION THREAD

**Discussion by philmetzger** [wevm/wagmi#695] [Q&A] [answered]
Hello all!

Trying out wagmi in my NextJS app but I get the following error:

`TypeError: (0 , wagmi__WEBPACK_IMPORTED_MODULE_12__.createClient) is not a function`

For this:

```
import { WagmiConfig, createClient } from 'wagmi'
import { getDefaultProvider } from 'ethers'

const client = createClient({
  autoConnect: true,
  provider: getDefaultProvider(),
})
```

This came straight off the [wagmi homepage](https://wagmi.sh/). Any ideas?

**Comment by philmetzger** [comment, marked as answer]
Ah i was calling createClient within my component!

**Comment by ghost** [comment]
problem not solved

**Reply by KingZedPriest** [reply]
Make that component a client component by adding "use client" at the top of the component. That solved it for me
